schemas/fingerprint-server-api.yaml

```diff
@@ -1410,36 +1409,33 @@ components:
           maximum: 1
     RawDeviceAttributesResult:
       type: object
+      properties:
```

Comment:
I don't understand, isn't `error` now part of the product like `rawDeviceAttributes.error` ? I thought it was supposed to be `rawDeviceAttribues.someKey.error` so part of the additional properties below, no?

Reply:
Ah, I that's a mistake from my side 😅. Should be fixed now!

Reply:
Better now :)